arkane/encorr/data.py

```diff
@@ -384,6 +390,16 @@
         'N#N': -2.0, 'O=O': -0.2, 'H-H': 1.1,  # Unknown source
     },
 
+    'g4': {
```

Comment:
Thanks! I appreciate this contribution.
Could you please add a brief comment of the data source used to derive these values?

Reply:
will do